packages/paste-core/primitives/combobox/stories/mutliselect.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import {Box} from '@twilio-paste/box';
+import {Input} from '@twilio-paste/input';
+import {Label} from '@twilio-paste/label';
+import {FormPill, FormPillGroup, useFormPillState} from '@twilio-paste/form-pill-group';
+import {ComboboxListbox, ComboboxListboxGroup, ComboboxListboxOption} from '@twilio-paste/combobox';
+
+import type {StoryFn} from '@storybook/react/dist/ts3.9/client/preview/types-6-0';
+
+import {ComboboxPrimitive, useComboboxPrimitive, useMultiSelectPrimitive} from '../src';
+
+const items = ['Alert', 'Anchor', 'Button', 'Card', 'Heading', 'List', 'Modal', 'Paragraph'];
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Primitives/Combobox',
+  component: ComboboxPrimitive,
+};
+
+const {FunctionAddSelectedItem, FunctionRemoveSelectedItem} = useMultiSelectPrimitive.stateChangeTypes;
+
+export const BasicMultiCombobox: StoryFn = () => {
+  const seed = useUIDSeed();
+  const [filteredItems, setFilteredItems] = React.useState([...items].sort());
+
+  const formPillState = useFormPillState();
+
+  const {
+    getSelectedItemProps,
+    getDropdownProps,
+    addSelectedItem,
+    removeSelectedItem,
+    selectedItems,
+  } = useMultiSelectPrimitive({
+    onSelectedItemsChange: ({type, selectedItems: currentSelectedItems}) => {
+      if (type === FunctionAddSelectedItem) {
+        const [toRemove] = difference(currentSelectedItems, selectedItems);
+        setFilteredItems((state) => state.filter((item) => item !== toRemove));
+      } else if (type === FunctionRemoveSelectedItem) {
+        const [toAdd] = difference(selectedItems, currentSelectedItems);
+        setFilteredItems((state) => [...state, toAdd as string].sort());
+      }
+    },
+  });
+
+  const {
+    getComboboxProps,
+    getInputProps,
+    getItemProps,
+    getLabelProps,
+    getMenuProps,
+    getToggleButtonProps,
+    highlightedIndex,
+    isOpen,
+    selectItem,
+    inputValue,
+  } = useComboboxPrimitive({
+    items: filteredItems,
+    initialInputValue: '',
+    onSelectedItemChange: ({selectedItem: selected}) => {
+      if (selected != null) {
+        addSelectedItem(selected);
+      }
+      // ℹ️ This action from the dropdown library is strongly typed to accept only a string
+      //   to reset, we need to pass "null" so we will cast it as a string e.g. "null as string".
+
+      // ℹ️ Downshift does not clear the "selectedItem" state after a selection has changed
+      //   with the multiple selection implementation, we need to force this case, so that if
+      //   we remove the item, it isn't still marked as "Selected" in the combobox state.
+      selectItem(null as string);
+    },
+  });
+
+  const dropdownProps = getDropdownProps({
+    preventKeyAction: isOpen,
+    ...getToggleButtonProps({tabIndex: 0}),
+  });
+
+  return (
+    <>
+      <Box marginBottom="space40" position="relative">
+        <Label htmlFor={seed('multi-select-primitive-input')} {...getLabelProps()}>
```

Comment:
Would using `seed('multi-select-primitive-input')` for htmlFor on the label and for the id of the input generate 2 different ids?